packages/@okta/vuepress-site/docs/guides/deploy-to-production/javascript/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: JavaScript
+---
+
+JavaScript applications, when built for production, are just a bunch of static files. You can deploy them to any web server, configure that web server to be SPA-aware, and you should be good to go!
+
+### Heroku
+
+One easy way to deploy your JavaScript app to production with Okta is with Heroku.
+
+To begin, install the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) and run `heroku login`.
+
+You can deploy your JavaScript application to Heroku in 5 steps:
+
+1. Run `heroku create`
+2. Add the Git remote that's created as a remote for your project
+
+       git remote add heroku <heroku-repo>
```

Comment:
should this be in tics? `git remove add heroku <heroku-repo>`

Also - steps 1 and 2 need periods.

Reply:
It seems to format it correctly as long as it's indented enough characters.